examples/node_refs/src/input.rs

```diff
@@ -7,6 +7,7 @@ pub enum Msg {
 #[derive(Properties, PartialEq)]
 pub struct Props {
     pub on_hover: Callback<()>,
+    pub placeholder: String,
```

Comment:
Optional: should use `AttrValue` instead of `String`.

Reply:
oh okay. I didn't know about that one. Had seen `String` in the docs example.

Reply:
updated, please review again